YouVsVirus/Assets/Scenes/levelsupermarket/AlreadyWasHere.cs

```diff
@@ -0,0 +1,23 @@
+﻿using UnityEngine;
+
+/// <summary>
+/// Attach to a place to make sure that player can
+/// go there only once
+/// </summary>
+public class AlreadyWasHere : MonoBehaviour
+{
+   public bool wasHere { get; private set; }
```

Comment:
```suggestion
    public bool wasHere { get; private set; }
```

Reply:
Better make this private and add a getter function?

Otherwise i can just change it from the outside and i.e. set it to false again after i was there.

Reply:
I am not sure if I totally understand the concept, but in I think the set function is private like here
https://codeasy.net/lesson/properties
See the example for
 class Student
  {
    public int Age { get; private set; }
  }
?